src/main/kotlin/no/nav/personbruker/dittnav/metrics/periodic/reporter/config/ApplicationContext.kt

```diff
@@ -39,34 +39,46 @@ class ApplicationContext {
 
     val kafkaMetricsReporter = TopicMetricsReporter(metricsReporter, nameScrubber)
 
-    val beskjedCountConsumer =
-        createCountConsumer<GenericRecord>(EventType.BESKJED, Kafka.beskjedTopicName, environment)
-    val innboksCountConsumer =
-        createCountConsumer<GenericRecord>(EventType.INNBOKS, Kafka.innboksTopicName, environment)
-    val oppgaveCountConsumer =
-        createCountConsumer<GenericRecord>(EventType.OPPGAVE, Kafka.oppgaveTopicName, environment)
-    val doneCountConsumer = createCountConsumer<GenericRecord>(EventType.DONE, Kafka.doneTopicName, environment)
-    val beskjedCounter =
-        TopicEventTypeCounter(beskjedCountConsumer, EventType.BESKJED, environment.deltaCountingEnabled)
-    val innboksCounter =
-        TopicEventTypeCounter(innboksCountConsumer, EventType.INNBOKS, environment.deltaCountingEnabled)
-    val oppgaveCounter =
-        TopicEventTypeCounter(oppgaveCountConsumer, EventType.OPPGAVE, environment.deltaCountingEnabled)
+    val beskjedKafkaProps = Kafka.counterConsumerProps(environment, EventType.BESKJED)
+    val oppgaveKafkaProps = Kafka.counterConsumerProps(environment, EventType.OPPGAVE)
+    val innboksKafkaProps = Kafka.counterConsumerProps(environment, EventType.INNBOKS)
+    val doneKafkaProps = Kafka.counterConsumerProps(environment, EventType.DONE)
+
+    var beskjedCountConsumer = initializeCountConsumer(beskjedKafkaProps, Kafka.beskjedTopicName)
+    var innboksCountConsumer = initializeCountConsumer(innboksKafkaProps, Kafka.innboksTopicName)
+    var oppgaveCountConsumer = initializeCountConsumer(oppgaveKafkaProps, Kafka.oppgaveTopicName)
+    var doneCountConsumer = initializeCountConsumer(doneKafkaProps, Kafka.doneTopicName)
+
+    val beskjedCounter = TopicEventTypeCounter(beskjedCountConsumer, EventType.BESKJED, environment.deltaCountingEnabled)
+    val innboksCounter = TopicEventTypeCounter(innboksCountConsumer, EventType.INNBOKS, environment.deltaCountingEnabled)
+    val oppgaveCounter = TopicEventTypeCounter(oppgaveCountConsumer, EventType.OPPGAVE, environment.deltaCountingEnabled)
     val doneCounter = TopicEventTypeCounter(doneCountConsumer, EventType.DONE, environment.deltaCountingEnabled)
+
     val topicEventCounterService = TopicEventCounterService(
-        beskjedCounter = beskjedCounter,
-        innboksCounter = innboksCounter,
-        oppgaveCounter = oppgaveCounter,
-        doneCounter = doneCounter
+            beskjedCounter = beskjedCounter,
+            innboksCounter = innboksCounter,
+            oppgaveCounter = oppgaveCounter,
+            doneCounter = doneCounter
     )
 
     val metricsSubmitterService = MetricsSubmitterService(
-        dbEventCounterService,
-        topicEventCounterService,
-        dbMetricsReporter,
-        kafkaMetricsReporter
+            dbEventCounterService,
```

Comment:
Hva slags autoformaterings oppsett er det du bruker? Det kan se ut som at vi bruker litt forskjellig i teamet🙂

Reply:
Trodde jeg hadde default. Er det noe spesielt jeg burde endre? 🙈

Reply:
<img width="563" alt="Skjermbilde 2021-02-04 kl  14 07 15" src="https://user-images.githubusercontent.com/37249421/106896822-5fc81900-66f2-11eb-9fe7-004e77f68c46.png">

Reply:
Oppsettet ser likt ut som det jeg har, men for filen `TopicEventCounterService` får jeg et annet resultat enn deg. I filen `ApplicationContext` får jeg samme resultat som deg.

Jeg begynte bare å lure siden det var så mange steder det var fikset på innrykk. Vi har jo egentlig aldri pratet skikkelig om autoformateringsregler, så vi kan sikkert ta en liten runde om det på Teknisk DittNAV snart😃